.gitignore

```diff
@@ -6,3 +6,4 @@ storage
 pid
 build/*
 tmp
+go-webapp
```

Comment:
Why you need this on gitignore?

Reply:
this is binary that gets built, because I can then work by loading this binary in docker.